api/src/main/java/com/codeforcommunity/rest/subrouter/TeamsRouter.java

```diff
@@ -22,12 +26,71 @@ public TeamsRouter(ITeamsProcessor processor) {
   @Override
   public Router initializeRouter(Vertx vertx) {
     Router router = Router.router(vertx);
-
+    registerCreateTeam(router);
+    registerGetTeam(router);
+    registerAddGoal(router);
+    registerDeleteGoal(router);
     registerDisbandTeam(router);
 
     return router;
   }
 
+  // Create a Team
+  private void registerCreateTeam(Router router) {
+    Route createTeamRoute = router.post("/create");
+    createTeamRoute.handler(this::handleCreateTeamRoute);
+  }
+
+  private void handleCreateTeamRoute(RoutingContext routingContext) {
+    JWTData userData = routingContext.get("jwt_data");
+    CreateTeamRequest createTeamRequest =
+        RestFunctions.getJsonBodyAsClass(routingContext, CreateTeamRequest.class);
+    processor.createTeam(userData, createTeamRequest);
+    end(routingContext.response(), 200);
+  }
+
+  // Get a Team
+  private void registerGetTeam(Router router) {
+    Route getTeamRoute = router.get("/:team_id");
+    getTeamRoute.handler(this::handleGetTeamRoute);
+  }
+
+  private void handleGetTeamRoute(RoutingContext routingContext) {
+    JWTData userData = routingContext.get("jwt_data");
+    GetTeamRequest getTeamRequest =
+        RestFunctions.getJsonBodyAsClass(routingContext, GetTeamRequest.class);
```

Comment:
Because the teamId is in the URL instead of in a JSON body in the spec use the getRequestParameterAsInt method in RestFunctions. The name for the route is the same thing as in line 54, so you would pass "team_id"